mysql/const.go

```diff
@@ -55,6 +55,11 @@ const (
 	ServerPSOutParams              uint16 = 0x1000
 )
 
+// IsCursorExists return true if cursor exists indicated by server status
+func IsCursorExists(serverStatus uint16) bool {
```

Comment:
We have functions like this in this form:
```
func HasCursorExistsFlag(serverStatus uint16) bool {
    serverStatus&ServerStatusCursorExists > 0
}
```

And I think this function is too simple to write a test for it.

Reply:
It has been updated as required.